features/filesystem/sd/SDBlockDevice.cpp

```diff
@@ -477,6 +509,57 @@ int SDBlockDevice::_cmd8() {
     return -1; // timeout
 }
 
+/* SDBlockDevice::_go_idle_state()
```

Comment:
Shall this live in the header file (that's where a declaration is). doxygen format? (might not be related as the rest of the private methods follow this). would be nice though,